templates/frontend/components/footer.tpl

```diff
@@ -15,19 +15,19 @@
 <footer class="main-footer" id="immersion_content_footer">
 	<div class="container">
 		{if $hasSidebar}
-			<div class="sidebar_wrapper row">
+			<div class="row">
 				{call_hook name="Templates::Common::Sidebar"}
 			</div>
 		{/if}
-		<div class="additional-footer-info row">
+		<div class="row">
```

Comment:
.additional-footer-info class that is deleted here also has associated style which makes alignment for footer content, including block plugins that are coming from the hook: justify-content: space-between;

Reply:
is this a class that comes from the default OJS theme?

Reply:
Nope. It's from Pragma

Reply:
Changed it to `main-footer__block` and pushed it. Will style in a separate PR!